src/main/java/EvenSymbolsNumberChecker.java

```diff
@@ -0,0 +1,50 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+
+public class EvenSymbolsNumberChecker {
+
+
+    public static HashSet<Character> symbolsChecker (ArrayList<String> str) {
+        int counter = 0;
+        int index = 0;
+        HashSet<Character> chars = new HashSet<>();
+        while (counter < 2 && index < str.size()) {
+            String stringFromArray = str.get(index);
+            Map<Character, Integer> letters = countCharacters(stringFromArray);
+
+            HashSet<Character> evenSymbols = checkAllSymbolsEven(letters);
```

Comment:
I think that better return type for `checkAllSymbolsEven` is `boolean` and rename it to `isAllSymbolsEven`, put this to if condition and add to `chars` like this:
```
if(isAllSymbolsEven(letters)) {
   chars.addAll(letters.keySet());
   counter += 1;
}
```